File: sample-infrastructure-persistence-orm/npm/src/typeorm/aggregates/parent/parent-schema.ts
import { EntitySchema } from "typeorm";
import { Parent } from "sample-domain";
import { EntityBaseSchemaColumns } from "../../abstractions/entity-schema";

export const ParentSchema = new EntitySchema<Parent>({
    name: "parent",
    columns: {
        ...EntityBaseSchemaColumns,
        description: {
            type: String
        }
    },
    relations: {
        children: {
            type: "one-to-many",
            target: "child",
            inverseSide: "_parent",
            cascade: true,
            eager: true
        }
    }
});

File: sample-infrastructure-persistence-orm/dotnet/src/nhibernate/aggregates/parent/parent-schema.cs
using NHibernate.Mapping.ByCode;
using NHibernate.Mapping.ByCode.Conformist;
using ParentAggregate = Sample.Domain.Aggregates.Parent;

namespace Sample.Infrastructure.Persistence.ORM.NHibernate.Aggregates.Parent
{
    public class ParentSchema : ClassMapping<ParentAggregate.Parent> 
    { 
        public ParentSchema()
        {
            Schema(NHSampleUnitOfWork.DEFAULT_SCHEMA);
            Table(nameof(ParentAggregate.Parent).ToLowerInvariant());
			Lazy(false);
            Id(p => p.Id, map => map.Generator(Generators.Increment));
            Property(p => p.Description);
            Set(
                p => p.Children, 
                c => {
                    c.Lazy(CollectionLazy.NoLazy);
                    c.Key(k => k.Column(nameof(ParentAggregate.Child.ParentId).ToLowerInvariant()));
                    c.Cascade(Cascade.All);
                }, 
                map => map.OneToMany(p => p.Class(typeof(ParentAggregate.Child)))
            );
        }
    }
}